frontend/src/components/Radio/useRadioGroup.ts

```diff
@@ -0,0 +1,22 @@
+import { createContext, RefObject, useContext } from 'react'
```

Comment:
chakra-ui itself has a `useRadioGroup` hook. Would this naming cause confusion?

Reply:
sure, renamed to `useRadioGroupWithOthers` in https://github.com/opengovsg/FormSG/pull/2628/commits/d5e266b58793355cc0cab8684821649c3d695793